linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/security/SSOUtils.scala

```diff
@@ -41,7 +41,7 @@ object SSOUtils extends Logging {
 
   private val sessionTimeout = ServerConfiguration.BDP_SERVER_WEB_SESSION_TIMEOUT.getValue.toLong
 
-  private val userTicketIdToLastAccessTime = new ConcurrentHashMap[String, Long]()
```

Comment:
It is recommended to instantiate it here to determine whether to use redis or the current way